src/views/registration-js/content.php

```diff
@@ -87,7 +94,7 @@
 						'event-tickets'
 					),
 					'<span class="tribe-tickets__notice--error__count">1</span>'
-			)
+				)
```

Comment:
:no_entry_sign: Each array item in a multi-line array declaration must end in a comma

<sub>hash: 6e2993f8c3ffac0b75bebce4630efe3c</sub>

Reply:
fixed